src/geode/geosciences/lso_input.cpp

```diff
@@ -153,16 +155,35 @@ namespace
             const auto tokens = get_tokens();
             for( const auto i : geode::Range{ 3 } )
             {
-                double value;
-                auto ok = absl::SimpleAtod( tokens[i + 2], &value );
-                OPENGEODE_EXCEPTION( ok, "[LSOInput] Error while "
-                                         "reading point coordinates" );
+                double value = geode::detail::read_double( tokens[i + 2] );
                 point.set_value( i, value );
             }
             point.set_value( 2, crs_.z_sign * point.value( 2 ) );
             return point;
         }
 
+        void read_point_properties( bool is_shared )
```

Comment:
I think these boolean are not great. Better having a read_shared_point_properties and a read_point_properties. Both could call another private method that takes the line_position as input. 

Ok it looks the same but this: read_point_properties( false ); is really bad for understanding the actual action of the method.